code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -512,6 +511,50 @@
 	M.adjust_bodytemperature(-8 * TEMPERATURE_DAMAGE_COEFFICIENT, BODYTEMP_NORMAL)
 	..()
 
+/datum/reagent/consumable/buzz_fuzz
+	name = "Buzz Fuzz"
+	description = "~A Hive of Flavour!~ NOTICE: Addicting."
+	id = "buzz_fuzz"
+	addiction_threshold = 14
+	color = "#8CFF00" // rgb: 135, 255, 0
+	taste_description = "carbonated honey and pollen"
+	glass_icon_state = "buzz_fuzz"
+	glass_name = "honeycomb of Buzz Fuzz"
+	glass_desc = "Stinging with flavour."
+
+/datum/reagent/consumable/buzz_fuzz/on_mob_life(mob/living/carbon/M)
+	M.reagents.add_reagent("sugar",5)
+	if(prob(50))
+		M.reagents.add_reagent("honey",2)
```

Comment:
jesus, 14u honey on average when below addiction threshold? that's a LOT of honey, since believe it or not adding more uses to honey doesn't make it less of a powergame heal chem

Reply:
wow, 14u, what was i on? no, it's 35

Reply:
Yep thats the main drawback

Reply:
5 x 5 = 25 base sugar + the 33% odds or 2 more sugar per honey making sugar coma a thing with this drink. Just like sugar rush

Reply:
The metabolic rate is standard (0.4u per tick) right? That means for one sip you get 55u sugar plus 2-22u honey, if I'm getting this right. 
`((25/6) / 0.4)*5`
`((5/6) / 0.4)*(0 or 2)`